apollo-annotations/build.gradle.kts

```diff
@@ -7,7 +7,17 @@ configureMppDefaults()
 kotlin {
   sourceSets {
     val commonMain by getting {
+      dependencies {
+        api(kotlin("stdlib", groovy.util.Eval.x(project, "x.versions.kotlinStdlib").toString()))
+      }
     }
+
+    val jsMain by getting {
+      dependencies {
+        api(kotlin("stdlib-js", groovy.util.Eval.x(project, "x.versions.kotlinStdlib").toString()))
```

Comment:
How come Gradle metadata doesn't pick this up?

Reply:
I don't know! But without it JS compilation complains - looks like a bug.